addons/calendar/static/src/views/fields/many2many_attendee_expandable.js

```diff
@@ -0,0 +1,66 @@
+/** @odoo-module **/
+
+import { useState, useEffect } from "@odoo/owl";
+import { registry } from "@web/core/registry";
+import { reposition } from "@web/core/position_hook";
+import {
+    Many2ManyAttendee,
+    many2ManyAttendee,
+    preloadMany2ManyAttendee,
+} from "@calendar/views/fields/many2many_attendee";
+
+export class Many2ManyAttendeeExpandable extends Many2ManyAttendee {
+    state = useState({ expanded: false });
+    shouldUpdatePopoverPosition = false;
+    attendeesCount = null;
+
+    setup() {
+        this.attendeesCount = this.props.record.data.attendees_count;
+
+        useEffect(() => {
+            if (this.shouldUpdatePopoverPosition) {
+                this.shouldUpdatePopoverPosition = false;
+
+                const popover = document.querySelector(".o_field_many2manyattendeeexpandable")
+                    .closest(".o_popover");
+                const targetElement = document.querySelector(`.fc-event[data-event-id="${this.props.record.resId}"]`);
+                reposition(targetElement, popover, { position: "right", margin: 0 });
+            }
+        });
```

Comment:
`useEffect` is a hook that is run on every re-render. It receives two parameters, a callback and a function that returns a list of dependencies. The dependencies are evaluated every re-render and if they change, the callback will be executed. So you can get rid of your `shouldUpdatePopoverPosition` property and you can just add `this.state.expanded` in your dependency array.

```suggestion
        useEffect(() => {
                const popover = document.querySelector(".o_field_many2manyattendeeexpandable")
                    .closest(".o_popover");
                const targetElement = document.querySelector(`.fc-event[data-event-id="${this.props.record.resId}"]`);
                reposition(targetElement, popover, { position: "right", margin: 0 });
            }
        }, () => [this.state.expanded]);
```